### DISCUSSION TITLE
Animation Direction

### DISCUSSION THREAD

**Discussion by CSpencerND** [dvtng/react-loading-skeleton#155] [Ideas]
Is there a way to change the animation direction to be vertical or diagonal (diagonal is what I really want). 
If not, I'd like to try implementing it myself if pointed in the right direction.

**Comment by srmagura** [comment]
I think this could be accomplished by tweaking [skeleton.css](https://github.com/dvtng/react-loading-skeleton/blob/c5dd39d03fa9296ef7e922ef268f47abd1d5d0a6/src/skeleton.css), specifically:

- Change the `translateX()`'s to `translate(x, y)`. The initial translation would be `(-50%, -50%)` and the final translation would be `(50%, 50%)`... or something like that.
- Change the direction of the `background-image: linear-gradient()` to align with the animation.

We can consider accepting your change into the library if it works well and does not add much complexity or bundle size.
